pkg/build/oci/oci.go

```diff
@@ -119,14 +129,50 @@ func buildImageFromLayer(layerTarGZ string, ic types.ImageConfiguration, created
 
 	v1Image, err = mutate.ConfigFile(v1Image, cfg)
 	if err != nil {
-		return empty.Image, fmt.Errorf("unable to update OCI config file: %w", err)
+		return nil, fmt.Errorf("unable to update OCI config file: %w", err)
 	}
 
-	return v1Image, nil
+	si := signed.Image(v1Image)
+
+	// Attach the SBOM, e.g.
+	if len(sbomFormats) > 0 {
+		var mt ggcrtypes.MediaType
+		var path string
+		switch sbomFormats[0] {
+		case "spdx":
+			mt = ctypes.SPDXMediaType
+			path = filepath.Join(sbomPath, fmt.Sprintf("sbom-%s.spdx.json", arch.ToAPK()))
```

Comment:
I wish the SBOM metadata were surfaced in a way we could plumb through to here, but the file list is effectively discarded today.  We should consider ways to improve the way this is plumbed through.